### PR TITLE
Including severity level on message description

### PR DESCRIPTION
**Pull request by GersonDias**
New PR message format:

**__{severity}__**: {Message} ({Rule ID})

### CODE REVIEW

File: src/module/prca/SonarQubeReportProcessor.ts [hunk 1]
```diff
@@ -19,6 +19,13 @@ import { ISonarQubeReportProcessor } from './ISonarQubeReportProcessor';
 export class SonarQubeReportProcessor implements ISonarQubeReportProcessor {
 
     private logger: ILogger;
+    private readonly priority_none = 6;
```
Comment (bgavrilMS) [inline src/module/prca/SonarQubeReportProcessor.ts]: Consider using an Enum instead https://www.typescriptlang.org/docs/handbook/enums.html

File: src/module/prca/SonarQubeReportProcessor.ts [hunk 2]
```diff
@@ -178,6 +190,23 @@ export class SonarQubeReportProcessor implements ISonarQubeReportProcessor {
         return message;
     }
 
+    private getSeverity(priority: Number): string {
```
Comment (bgavrilMS) [inline src/module/prca/SonarQubeReportProcessor.ts]: I'd rename this to "getSeverityDisplayName" or smth similar

File: src/module/prca/SonarQubeReportProcessor.ts [hunk 3]
```diff
@@ -160,6 +167,11 @@ export class SonarQubeReportProcessor implements ISonarQubeReportProcessor {
         let content: string = `${issue.message} (${issue.rule})`;
         let priority: number = this.getPriority(issue);
 
+        if (priority < this.priority_none) {
```
Comment (bgavrilMS) [inline src/module/prca/SonarQubeReportProcessor.ts]: This block can be moved to 'getPriority'

**Comment by GersonDias** [comment]
Hello Bogdan

I made a new commit with your suggestions.

**Comment by bgavrilMS** [comment]
Good stuff, thanks!

**Pull request merged**